src/components/search-input/search-input.js

```diff
@@ -15,7 +15,7 @@ export const SearchInput = ({
     if (value) {
       setCurrentValue(value);
     }
-  }, []);
+  }, [value]);
```

Comment:
The reason it was set to only update the `currentValue` from outside the component once is that we don't want to rely on a state from outside the component. So if we want to initialize the `currentValue`, we can, but any other change that comes after we rely on the internal state and then do the callback to notify parent components. Please let me know if this makes sense.
```suggestion
  }, []);
```